objectRecognitionSource/README.md

```diff
@@ -167,6 +167,12 @@ The Configuration document may look similar to the following example:
              "outputDir": "imageOut",
              "savedResolution": {
                 "longEdge": 400
+             },
+             "preCrop": {
+                "x": 50,
+                "y": 100,
+                "w": 600,
```

Comment:
`w` & `h` --> `width` & `height`?  unless we have other places where we use `w` & `h`